instrumentation/servlet/servlet-3.0/src/test/groovy/AbstractServlet3Test.groovy

```diff
@@ -88,6 +88,8 @@ abstract class AbstractServlet3Test<SERVER, CONTEXT> extends HttpServerTest<SERV
         "${SemanticAttributes.HTTP_URL.key()}" { it == "${endpoint.resolve(address)}" || it == "${endpoint.resolveWithoutFragment(address)}" }
         "${SemanticAttributes.HTTP_METHOD.key()}" method
         "${SemanticAttributes.HTTP_STATUS_CODE.key()}" endpoint.status
+        "${SemanticAttributes.HTTP_RESPONSE_CONTENT_LENGTH.key()}" { it == null || it == getContentLength(endpoint) }
```

Comment:
curious under what conditions is response content length not captured? (why we need `it == null`)

Reply:
@trask There are quite a few where response length is not captured in the tests. 
![image](https://user-images.githubusercontent.com/17055695/87994314-7afa7080-caa1-11ea-9be9-0532eb1fd130.png)

I haven't figured out how to track down why that is happening yet. The groovy tests are a lot less straightforward than junit.

Reply:
All of the sync tests are happy, maybe my async implementation is bad.

Reply:
checking the actual okhttp response body length in the test fixed the async tests